api/restore.go

```diff
@@ -51,43 +69,65 @@ func (c *Client) prepareRestore(opts *rpc.CriuOpts, args *task.RestoreArgs, chec
 	_, err = os.Stat(filepath.Join(tmpdir, "checkpoint_state.json"))
 	if err != nil {
 		c.logger.Fatal().Err(err).Msg("checkpoint_state.json not found, likely error in creating checkpoint")
-		return nil, nil, err
+		return nil, nil, nil, err
 	}
 
 	data, err := os.ReadFile(filepath.Join(tmpdir, "checkpoint_state.json"))
 	if err != nil {
 		c.logger.Fatal().Err(err).Msg("error reading checkpoint_state.json")
-		return nil, nil, err
+		return nil, nil, nil, err
 	}
 
 	var checkpointState task.ProcessState
 	err = json.Unmarshal(data, &checkpointState)
 	if err != nil {
 		c.logger.Fatal().Err(err).Msg("error unmarshaling checkpoint_state.json")
-		return nil, nil, err
+		return nil, nil, nil, err
 	}
 
-	// check open_fds. Useful for checking if process being restored
-	// is a pts slave and for determining how to handle files that were being written to.
-	// TODO: We should be looking at the images instead
 	open_fds := checkpointState.ProcessInfo.OpenFds
-	var isShellJob bool
+
+	// create logfile for redirection
+	file, err := os.Create("/var/log/cedana-output-restored.log")
```

Comment:
What happens if you c/r again on the restored process? It will log to the same restored log right?

Reply:
hmm yeah good point, maybe I'll throw a timestamp in

Reply:
fixed